File: src/java/1005.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in); 
        double a = in.nextDouble();
        double b = in.nextDouble();

        System.out.printf("MEDIA = %.5f\n", (a * 3.5 + b * 7.5) / 11);
    }
}

File: kotlin/1005.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`);
    
    val a: Double = sc.next().toDouble();
    val b: Double = sc.next().toDouble();
    
    println("MEDIA = ${((a * 3.5 + b * 7.5) / 11).format(5)}");
}

fun Double.format(digits: Int) = "%.${digits}f".format(this)